Packs/JamfProtect/Integrations/JamfProtectEventCollector/JamfProtectEventCollector.py

```diff
@@ -452,27 +599,36 @@ def fetch_events(client: Client, max_fetch_alerts: int, max_fetch_audits: int, s
         end_date_arg (str, optional): The end date for fetching events.
 
     Returns:
-        tuple[List[dict], dict]: A tuple containing two elements:
-            - A list of dictionaries. Each dictionary represents an event.
-            - A dictionary with the key 'last_fetch' and its value representing the end date of the fetched events.
+        EventResult: A NamedTuple containing four elements:
+            - alert_events: A list of dictionaries. Each dictionary represents an alert event.
+            - audit_events: A list of dictionaries. Each dictionary represents an audit event.
+            - computer_events: A list of dictionaries. Each dictionary represents a computer event.
+            - next_run: A dictionary with the keys 'alert', 'audit', 'computer', and optionally 'nextTrigger'.
+                    Each key (except 'nextTrigger') maps to a dictionary representing the next run details for that event type.
```

Comment:
I think this is a makes good documentation for the EventResult class, rather than here.